api/openapi-spec/openapi-spec/v3/.well-known__openid-configuration_openapi.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
All these files appear to have a doubled-up openapi-spec directory, perhaps you ran the generator with the wrong working directory?